src/chapters/classes/exercises.rst

```diff
@@ -0,0 +1,13 @@
+Exercises: Classes and Objects
+==============================
+
+Work on these exercises in the IntelliJ ``java-web-dev-exercises`` project.
+
+1. Open up the file, ``StudentClass.java``, and add all of the necessary getters and setters. 
+   Think about the access level each field and method should have, and try reducing the access level of at least one setter to less than public.
+2. In the ``classes1`` package, create a class ``Course`` with at least three fields.
+   Before diving into IntelliJ, try using pen and paper to work through what these might be.
+   At least one of your fields should be an ``ArrayList`` or ``HashMap``, and you should use your ``Student``
+   class.
+3. In the ``classes1`` package, create a class ``Teacher`` with four fields: ``firstName``, ``lastName``, ``subject``, and ``yearsTeaching``.
+   Add the appropriate getters and setters to the class and add the appropriate access level to each field and methd in the class.
```

Comment:
While "appropriate" seems OK for the getters and setters comment, it seems a little too vague to use with the access levels.  What, exactly, does "appropriate" mean?

Perhaps we could say, "...and set the access level for each field and method in the class," and then follow with some guiding questions for the students to consider.  This makes them think about how the data will be used and how open it should be to outside code.

Reply:
I think I came up with a good solution! Please let me know your thoughts!

Reply:
I like the change, it makes the students consider what to use instead of just randomly typing in ``public`` or ``private``.